src/components/projects/EditorElement.tsx

```diff
@@ -30,19 +30,21 @@ import { findParentNodeClosestToPos, flatten } from 'prosemirror-utils'
 import React, { useState } from 'react'
 import { useDrop } from 'react-dnd'
 
+import { PlaceholderWrapper } from '../../EditorApp'
 import { useConnectEditor } from '../../hooks/use-connect-editor'
 import { useWatchTitle } from '../../hooks/use-watch-title'
 import { setNodeAttrs } from '../../lib/node-attrs'
 import { useStore } from '../../store'
 import { SpriteMap } from '../track-changes/suggestion-list/Icons'
+import { ManuscriptPlaceholder } from './ManuscriptPlaceholder'
 
 const EditorElement: React.FC = () => {
   const [error, setError] = useState('')
   const [{ editor }] = useStore((store) => ({
     editor: store.editor,
   }))
 
-  const { onRender, view, dispatch } = useConnectEditor()
+  const { onRender, view, dispatch, isComparingDocuments } = useConnectEditor()
```

Comment:
If you will prepare the document with comparison during Sources assembly stage as suggested in my comment in [src/hooks/use-create-editor.ts](https://github.com/Atypon-OpenSource/manuscripts-article-editor/pull/841/files#diff-5bf36822dcc2e4e9202ab1b425ff38501155b2e69cbf3dab773289427b2ff035) in this PR, you won't have to create any special loading procedures in the body editor. That also makes sense because the loading is already managed there and there is no need to make another loading as it's the only purpose of this instance - to compare snapshots.